moveit_ros/planning/planning_request_adapter_plugins/CMakeLists.txt

```diff
@@ -1,13 +1,35 @@
 set(MOVEIT_LIB_NAME moveit_default_planning_request_adapter_plugins)
 
 set(SOURCE_FILES
-  src/empty.cpp
-  src/fix_start_state_bounds.cpp
-  src/fix_start_state_collision.cpp
-  src/fix_start_state_path_constraints.cpp
-  src/fix_workspace_bounds.cpp
-  src/add_time_parameterization.cpp
-  src/add_iterative_spline_parameterization.cpp)
+        src/empty.cpp
+        src/fix_start_state_bounds.cpp
+        src/fix_start_state_collision.cpp
+        src/fix_start_state_path_constraints.cpp
+        src/fix_workspace_bounds.cpp
+        src/add_time_parameterization.cpp
+        src/add_iterative_spline_parameterization.cpp
+        src/chomp_optimizer_adapter.cpp)
+
+find_package(catkin REQUIRED COMPONENTS
+        cmake_modules
+        roscpp
+        moveit_core
+        moveit_msgs
+        pluginlib
+        chomp_motion_planner
+        moveit_experimental
+        )
+
+find_package(Eigen3 REQUIRED)
+
+
+include_directories(../../../moveit_planners
+        ../../../moveit_planners/chomp/chomp_motion_planner/include/chomp_motion_planner
+        ../../../moveit_planners/chomp/chomp_motion_planner/include/
+        ../../../moveit_planners/chomp/chomp_motion_planner/
+        ../../../moveit_experimental/collision_distance_field/include/moveit/collision_distance_field
+        ../../../moveit_experimental/collision_distance_field/include/
+        ../../../moveit_experimental/collision_distance_field/)
```

Comment:
this is a bad way to include directories

Reply:
I agree this is bad, could you tell me whats the correct way to include these? I tried `find_package (chomp_motion_planner)` and `find_package(moveit_experimental)` and included these (only locally, did not commit, as it didn't work) as a dependency in the [`target_link_libraries`](https://github.com/raghavendersahdev/moveit/blob/CHOMP_planning_request_adapter/moveit_ros/planning/planning_request_adapter_plugins/CMakeLists.txt#L32). Currently I got rid of 4 redundant ones [here](https://github.com/ros-planning/moveit/pull/1012/commits/9db5761d4b3054589eb940278aa87f504ab01df9#diff-35fcbb8c15ffab0b4c7ac6b0db7154deR25)